src/ui/RequestContainer.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <li class="campaign-card">
+    <figure>
+      <div>
+        <img :src="campaign.image"/>
+      </div>
+      <figcaption>
+        <h3>{{ campaign.name }}</h3>
+        <p>{{campaign.description}}</p>
+        <ul v-for="request in campaign.requests" v-bind:key="request.id">
+          <li>
+            <p><b>Summary:</b>{{request.description}}</p>
+            <p><b>Value:</b>{{request.value}}</p>
+            <button v-on:click="$emit('approval-count',
+              {address: campaign.address, ethId: request.eth_id})">
+              Confirm emitted events
+            </button>
+            <button v-on:click="$emit('approve-request', {
+              address: campaign.address,
+              ethId: request.eth_id,
+              requestID: request.id})"
+            >
+              ApproveRequests
+            </button>
+          </li>
+        </ul>
+      </figcaption>
+    </figure>
+  </li>
+</template>
```

Comment:
Basic structure of new Requests To Approve card, will grow in complexity as it grow in modularity. But I think it can be done.

Reply:
Nice! I guess this is better than having a div with an empty string class name....

Reply:
I'm so impressed by the use of $emit and taking in the destructured object